metrics/lib/send_results.sh

```diff
@@ -0,0 +1,424 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# This script will store the results into csv files in the
+# metrics 'results' directory.
+# The csv file names are derived from the reported 'test name', and all
+# results for a single 'test name' are collated (appended) into a single csv.
+
+# General env
+MYSELF="${0##*/}"
+SCRIPT_PATH=$(dirname $(readlink -f $0))
+source "${SCRIPT_PATH}/common.bash"
+
+# Override the RESULT_DIR from the test-common.bash for the moment.
+# Once all all tests are migrated to use this script, remove the
+# RESULT_DIR definition from test-common.bash, as all results should
+# be then stored via this script
+RESULT_DIR="${SCRIPT_PATH}/../results"
+
+GROUP="PNP"
+CC_DEFAULT_RUNTIME_PATH="/usr/bin/cc-oci-runtime"
+RUNC_DEFAULT_RUNTIME_PATH="/usr/bin/docker-runc"
+CC_RUNTIME_DEFAULT_PATH="/usr/local/bin/cc-runtime"
+CC_DEFAULT_IMG_PATH="/usr/share/clear-containers"
+CONTAINERS_IMG_SYSTEM="/usr/share/clear-containers/clear-containers.img"
+CONTAINERS_IMG_GUESS="${CC_DEFAULT_IMG_PATH}/clear-containers.img"
+CONTAINER_KERNEL_SYSTEM="/usr/share/clear-containers/vmlinux.container"
+CONTAINER_KERNEL_GUESS="${CC_DEFAULT_IMG_PATH}/vmlinux.container"
+SYSTEM_CLEAR_RELEASE_FILE="/usr/lib/os-release"
+SYSTEM_CENTOS_RELEASE_FILE="/etc/os-release"
+
+declare TEST_NAME
+declare RESULT
+declare ARGS
+declare UNITS
+declare SYS_VERSION
+declare SYSTEM
+declare TAG
+declare HW
+declare SEND
+declare KERNEL
+declare IMG
+
+# Parse the platform name
+function get_platform_name()
+{
+	com="core"
+
+	model="$(cat /proc/cpuinfo | grep "model name" | uniq | \
+		awk -F ": " '{print $2}' | sed s/\(R\)//g | \
+		sed s/\(TM\)//g | sed s/[[:space:]]*CPU[[:space:]]*/" "/g | \
+		cut -d"@" -f1 | sed s/[[:space:]]*$//g)"
+
+	num_cores=$(nproc)
+
+	if (( $num_cores > 1 ));then
+		com="cores"
+	fi
+
+	echo "$model ($num_cores $com)"
+}
+
+
+function get_cor_version()
+{
+	# If docker could tell us the path to the actual runtime then
+	# this is where we would grab and use that info.
+
+
+	# See if runtime is in the default place
+	if [ -f "${CC_DEFAULT_RUNTIME_PATH}" ];then
+		runtime="${CC_DEFAULT_RUNTIME_PATH}"
+	else
+		# And if we cannot find it at the default, can we find it
+		# in the path
+		runtime="$(which cc-oci-runtime)"
```

Comment:
It would be nice if:

```
CC_DEFAULT_RUNTIME="cc-oci-runtime"
CC_DEFAULT_RUNTIME_PATH="/usr/bin/${CC_DEFAULT_RUNTIME}"
```

Then, you could say `runtime=$CC_DEFAULT_RUNTIME` here.